core/frontend/src/components/version-chooser/VersionChooser.vue

```diff
@@ -330,6 +330,12 @@ export default Vue.extend({
       if (!this.current_version) {
         return ''
       }
+      try {
+        const test = new SemVer(this.current_version.tag)
+      } catch {
+        // if we are on a random, non-SemVer tag, we cant compare it to a SemVer tag.
+        return ''
+      }
```

Comment:
The compiler seems skeptical about the need for this try-catch

Reply:
got rid of it